apps/etterlatte-behandling/src/main/kotlin/oppgaveGosys/GosysOppgaveService.kt

```diff
@@ -83,23 +87,58 @@ class GosysOppgaveServiceImpl(
             .expireAfterWrite(Duration.ofMinutes(5))
             .build<Long, GosysOppgave>()
 
+    private fun hentEnehterForSB(
```

Comment:
```suggestion
    private fun hentEnheterForSB(
```
evt. `hentEnheterForSaksbehandler`. 

Men jeg henger ikke helt med på hvorfor denne ikke validerer enhet vi sender inn opp mot de enhetene saksbehandler har